collections.go

```diff
@@ -6,27 +6,25 @@ import "time"
 type Collections []Collection
 
 // Collection is a grouping of products.
-type Collection struct {
-	// ID is the ID for the collection.
-	ID int64
-	// Title is the name of the collection. (limit: 255 characters)
-	Title string
+type Collection interface {
```

Comment:
the fields in here should be in alphabetical order, need to swap image and id and move product count to the correct place

Reply:
I thought we are displaying them from what the shopify api docs says?

Reply:
we are, which I believe are alphabetical, either way image and id are the wrong way around